### PR TITLE
193 los tweaks

### PR DESCRIPTION
**Pull request by mvl22**
All fixes possible, pending new data, requested in #193.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -237,20 +237,20 @@ <h2>Layer controls</h2>
 			<button class="accordion">Coherent network</button>
 			<div class="panel">
 				<p>Indicates a suggested order in which the network should grow.</p>
-				<p><label><input type="checkbox" class="showlayer" data-layer="cohesivenetwork" value="false"> Show cohesive network</label></p>
-				<div id="cohesivenetworklegend" class="legend"></div>
+				<p><label><input type="checkbox" class="showlayer" data-layer="coherentnetwork" value="false"> Show coherent network</label></p>
```
Comment (Robinlovelace) [inline index.html:240]: Yep, matching #194

File: index.html [hunk 2]
```diff
@@ -237,20 +237,20 @@ <h2>Layer controls</h2>
...
+				<div id="coherentnetworklegend" class="legend"></div>
 			</div>
 			
-			<button class="accordion">Cycling by design compliance</button>
+			<button class="accordion">Cycling by Design compliance</button>
```
Comment (Robinlovelace) [inline index.html:244]: :+1: although I wonder: should we drop the rather superfluous "compliance" bit:

```suggestion
			<button class="accordion">Cycling by Design</button>
```

File: index.html [hunk 3]
```diff
@@ -237,20 +237,20 @@ <h2>Layer controls</h2>
...
 			<div class="panel">
-				<p>Data on existing infrastructure, speed limits, volumes and estimated Cycling Level of Service (LoS).</p>
+				<p>Data on existing infrastructure, speed limits, volumes and estimated Cycling Level of Service (LoS), as per the <a href="https://www.transport.gov.scot/media/50323/cycling-by-design-update-2019-final-document-15-september-2021-1.pdf" target="_blank" title="[Link opens in a new window]">Cycling by Design guidance</a>. <button aria-label="Help on LoS" class="helpbutton" data-help="clos"><i class="fa fa-question-circle" aria-hidden="true"></i></button></p>
```
Comment (Robinlovelace) [inline index.html:246]: This looks like a great explanation, good job!

File: index.html [hunk 4]
```diff
@@ -237,20 +237,20 @@ <h2>Layer controls</h2>
...
 				<p><label><input type="checkbox" class="showlayer" data-layer="clos" value="false"> Show LoS</label></p>
 				<p>
 					<select name="clos-layer" class="updatelayer" data-layer="clos" aria-label="LoS detail">
 						<option value="Level of Service" selected="selected">Current LoS</option>
 						<option value="Traffic volume">Current traffic volume</option>
 						<option value="Speed limit">Current speed limit</option>
-						<option value="Infrastructure type">Existing infrastructure</option>
+						<option value="Infrastructure type">Physical cycle infrastructure</option>
```
Comment (Robinlovelace) [inline index.html:253]: :+1:

**Review by Robinlovelace — APPROVED** [review]

File: index.html (hunk 2 above)
Comment (mvl22) [reply index.html:244]: Well I think the point is that the data is intended to show the level of compliance compared with CbD. So compliance presumably necessary.

Comment (Robinlovelace) [reply index.html:244]: The thing is: it shows lots of other things also, not just compliance. But happy with it as is for now.

Comment (mvl22) [reply index.html:244]: Yes, fair point.

**Pull request merged**